app.py

```diff
@@ -38,7 +38,48 @@ def mirror(name):
     }
     return create_response(data)
 
-# TODO: Implement the rest of the API here!
+@app.route('/users', methods = ['GET', 'POST'])
+def users():
+    if request.method == 'GET':
+        if request.args.get('team') is None:
+            data = {
+                'users': db.get('users')
+            }
+        else:
+            matched = [i for i in db.get('users') if i['team'] == request.args.get('team')]
+            data = {
+                'users': matched
+            }
```

Comment:
While this works, this part can be condensed down by not rewriting how you define `data` and calling `request.args.get('team')` twice e.g.:
```
team = request.args.get('team')
data =db.get('users')
if team is None:
    matched = [i for i in data if i['team'] == team]
    return create_response({'users': matched})
return create_response({'users': data})
```